.env.example

```diff
@@ -3,6 +3,9 @@ SEPOLIA_RPC_URL=
 MAINNET_RPC_URL=
 
 PRIVATE_KEY=""
```

Comment:
Could you remove this and rename DEPLOYER to DEPLOYER_PRIVATE_KEY? And then replace use of PRIVATE_KEY that too